config/7.1.0/securebanking/ig/routes/routes-service/30-ob-payment-consent-funds-confirmation.json

```diff
@@ -211,7 +211,7 @@
             "args": {
               "auditService": "${heap['AuditService-OB-Consent']}",
               "clock": "${heap['Clock']}",
-              "consentIdLocator": "response.entity.getJson().OBIntentObject.Data.ConsentId",
+              "consentIdLocator": "new groovy.json.JsonSlurper().parseText(contexts.oauth2.accessToken.info.claims).id_token.openbanking_intent_id.value",
```

Comment:
this configuration is used by the accounts routes when accessing an account resource.